### DISCUSSION TITLE
front-camera mirror effect

### DISCUSSION THREAD

**Discussion by ajedari** [ant-media/Ant-Media-Server#5669] [Q&A]
Does ant-media provide facility to handle mirror effect for front-camera ?

**Comment by yashtandon113** [comment]
Hi @ajedari 
Thanks for your question.
Could you kindly provide additional details about the issue you're having and what you would like to accomplish?

**Reply by ajedari** [reply]
Thanks for quick response @yashtandon113,
Sure, I have mentioned scenario below :
When I do live streaming with end persons, they facing mirror effect that I want to prevent. Also I have facility to upload live stream as a post in my ionic/react app but when I post stream as a video file to end users. They facing mirror effect with video as well.

**Reply by yashtandon113** [reply]
Thank you for providing the information.
Could you please tell me which Ant Media SDK you're using? Because there is no such problem on the AMS sample publishing page that uses javascript.

It would good and time saving for both of us, if you can provide the step by step scenario as per below template:

**Publishing protocol:** WebRTC/RTMP/RTSP
**Playing protocol:** WebRTC/RTMP/HLS

**Steps to reproduce**
1.
2.
3.

**Expected behavior**
Put as much detail here as possible

**Actual behavior**
Put as much detail here as possible

**Comment by alignwebs** [comment]
@ajedari were you able to fix it?
